centreon/www/front_src/src/ResourceAccessManagement/Actions/Delete/DeleteConfirmationDialog.tsx

```diff
@@ -33,11 +36,20 @@ const DeleteConfirmationDialog = (): JSX.Element => {
       labelCancel={t(labelCancel)}
       labelConfirm={t(labelDelete)}
       labelMessage={
-        resourceAccessRuleName &&
-        `The ${resourceAccessRuleName} ${t(labelDeleteResourceAccessRuleDialogMessage)}`
+        resourceAccessRuleName
+          ? `The ${resourceAccessRuleName} ${t(labelDeleteResourceAccessRuleDialogMessage)}`
+          : t(labelDeleteResourceAccessRulesDialogMessage)
+      }
+      labelSecondMessage={
+        resourceAccessRuleName
+          ? t(labelDeleteResourceAccessRuleWarning)
+          : t(labelDeleteResourceAccessRulesWarning)
+      }
+      labelTitle={
+        resourceAccessRuleName
+          ? t(labelDeleteResourceAccessRule)
+          : t(labelDeleteResourceAccessRules)
```

Comment:
you may extract this logic into separated variables

Reply:
Done